themes/adminimal_medela/build/js/publication-rules.js

```diff
@@ -0,0 +1,69 @@
+/**
```

Comment:
This file should be located in the `themes/adminimal_medela/source/js/` directory so that it can be minified and added to the build/ directory by Gulp.